electric-operator.el

```diff
@@ -225,7 +231,9 @@ Whitespace before the operator is captured for possible use later.
       (let* ((pre-whitespace (match-string 1))
              (op-match-beginning (match-beginning 0))
              (op-match-end (match-end 0))
-             (spaced-string (eval-action action op-match-beginning)))
+             (op-start-pos (+ op-match-beginning (length pre-whitespace)))
+             (state `((start-pos . ,op-start-pos)))
```

Comment:
As much as I dislike how verbose emacs' defstruct stuff is", I think this should probably be one so that people adding functions can have a nicer interface than `caar` etc.